x-pack/test/functional_endpoint/apps/endpoint/index.ts

```diff
@@ -9,12 +9,11 @@ export default function ({ loadTestFile }: FtrProviderContext) {
   describe('endpoint', function () {
     this.tags('ciGroup7');
 
-    loadTestFile(require.resolve('./feature_controls'));
-    loadTestFile(require.resolve('./landing_page'));
-    loadTestFile(require.resolve('./header_nav'));
-    loadTestFile(require.resolve('./host_list'));
+    loadTestFile(require.resolve('./endpoint_list'));
     loadTestFile(require.resolve('./policy_list'));
-    loadTestFile(require.resolve('./alerts'));
-    loadTestFile(require.resolve('./resolver'));
+    loadTestFile(require.resolve('./policy_details'));
+
+    // loadTestFile(require.resolve('./alerts'));
```

Comment:
small nit, we could add a TODO or just remove the commented out code and track enabling the rest of the tests in another ticket

Reply:
Good idea. I will remove the commented out code in a subsquent PR and will create an Issue to follow up on it (I don't think these are applicable anymore)